manifest.json

```diff
@@ -10,6 +10,11 @@
         "arch": "x86_64-darwin",
         "checksum": "fdbf094e07fb2133d7ee58f2bacdca2dd7b112bcf9b9c078abb496f845f72f5a"
       },
+      {
+        "url": "https://github.com/LuaLS/lua-language-server/releases/download/3.6.25/lua-language-server-3.6.25-darwin-arm64.tar.gz",
+        "arch": "aarch64-darwin",
+        "checksum": "SKIP"
```

Comment:
You can try with `shasum -a 256 filename` or `openssl sha256 filename`.

Reply:
Nice! Thank you